### PR TITLE
Quote banner and introduce paragraph

### PR DESCRIPTION
**Pull request by hoanguyennph**

### CODE REVIEW

**Comment by LongDuongBIT** [comment]
@hoanguyennph You haven't got latest code from base branch.

File: VMS/Pages/Homepage/QuoteBanner.razor.scss [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+        position: absolute;
+        left: 10px;
+        top: 20px;
+        font-family: Montserrat;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 20px;
+        line-height: 24px;
+        color: $green;
+        border: 0.25px solid rgba(0, 0, 0, 0.2);
```
Comment (ThanhLamUit) [inline VMS/Pages/Homepage/QuoteBanner.razor.scss:31]: please delete this border

File: VMS/Resources/scss/common/_variables.scss [hunk 2]
```diff
@@ -7,12 +7,6 @@ $green: #64D72F;
 $green-bold: #306C14;
 $grey-bg: #E5E5E5;
 $grey-text: #515151;
-$greyicon: #B4B4B4;
-$grey: #F3F3F5;
-$blue-hv: #0B588c;
-$green-hv: #306C14;
-$grey-hv: #646464;
-
```
Comment (ThanhLamUit) [inline VMS/Resources/scss/common/_variables.scss:15]: these lines were deleted, please revert them

File: VMS/VMS.csproj [hunk 3]
```diff
@@ -43,7 +43,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="wwwroot\img\RelatedActivities\" />
```
Comment (ThanhLamUit) [inline VMS/VMS.csproj:46]: this line was deleted, please revert it

File: VMS/Pages/Homepage/QuoteBanner.razor.scss (hunk 1 above)
Comment (hoanguyennph) [reply VMS/Pages/Homepage/QuoteBanner.razor.scss:31]: Okay, i will fix it

File: VMS/Resources/scss/common/_variables.scss (hunk 2 above)
Comment (LongDuongBIT) [reply VMS/Resources/scss/common/_variables.scss:15]: I reverted them for @hoanguyennph

File: VMS/VMS.csproj (hunk 3 above)
Comment (LongDuongBIT) [reply VMS/VMS.csproj:46]: This line is somehow irrelevant, just ignore it.

File: VMS/Pages/Homepage/QuoteBanner.razor.scss (hunk 1 above)
Comment (LongDuongBIT) [reply VMS/Pages/Homepage/QuoteBanner.razor.scss]: I removed it.

File: VMS/VMS.csproj (hunk 3 above)
Comment (ThanhLamUit) [reply VMS/VMS.csproj:46]: i got it

**Review by ThanhLamUit — APPROVED** [review]

**Pull request merged**